### PR TITLE
Change apt-get to apt

### PR DESCRIPTION
**Pull request by JamesH65**
This is a global replace of apt-get with apt in the docs repo.

### CODE REVIEW

**Comment by JamesH65** [comment]
@XECDesign Seem ok to you?

**Comment by lurch** [comment]
There's a few places that mention `apt upgrade` - do any of them need changing to `apt dist-upgrade` ?

File: raspbian/updating.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The kernel and firmware are installed as a Debian package, and so will also get
 
 ### Running out of space
 
-When running `sudo apt-get dist-upgrade`, it will show how much data will be downloaded and how much space it will take up on the SD card. It's worth checking with `df -h` that you have enough free disk space, as unfortunately `apt` will not do this for you. Also be aware that downloaded package files (`.deb` files) are kept in `/var/cache/apt/archives`. You can remove these in order to free up space with `sudo apt-get clean`.
+When running `sudo apt dist-upgrade`, it will show how much data will be downloaded and how much space it will take up on the SD card. It's worth checking with `df -h` that you have enough free disk space, as unfortunately `apt` will not do this for you. Also be aware that downloaded package files (`.deb` files) are kept in `/var/cache/apt/archives`. You can remove these in order to free up space with `sudo apt clean`.
```
Comment (XECDesign) [inline raspbian/updating.md]: One difference in behaviour between `apt` and `apt-get` is that `apt` will remove the downloaded archives afterwards. So the last two sentences are not necessary.

**Comment by XECDesign** [comment]
> There's a few places that mention `apt upgrade` - do any of them need changing to `apt dist-upgrade` ?

I'm a fan of having a single place that tell you how to upgrade rather than every page starting with repeated information. However, any time I take out it out and replace it with a link to the updating page instead, somebody sends a PR that adds it back.

File: raspbian/updating.md (hunk 1 above)
Comment (lurch) [reply raspbian/updating.md]: Oooh, I never knew that :slightly_smiling_face:

Comment (JamesH65) [reply raspbian/updating.md]: Fixed

**Comment by JamesH65** [comment]
So, shall I change all to dist-upgrades?

**Comment by XECDesign** [comment]
Maybe `apt full-upgrade`? `dist-upgrade` is not a documented command of `apt`, it seems to be a backwards compatibility alias for `full-upgrade`

**Comment by XECDesign** [comment]
It was also avoid the confusion of people thinking `dist-upgrade` is just for upgrading between releases.

**Comment by JamesH65** [comment]
OK, will change to full-upgrade.

**Comment by XECDesign** [comment]
LGTM

**Pull request merged**